esp/bindings/http/client/httpclient.cpp

```diff
@@ -405,13 +405,134 @@ int CHttpClient::sendRequest(const char* method, const char* contenttype, String
     return 0;
 }
 
+void copyHeaders(CHttpMessage &copyTo, CHttpMessage &copyFrom)
+{
+    if (copyFrom.queryHeaders().ordinality())
+    {
+        ForEachItemIn(i, copyFrom.queryHeaders())
+        {
+            StringArray pair;
+            pair.appendList(copyFrom.queryHeaders().item(i), ":", true);
+            const char *name = pair.item(0);
+            const char *value = pair.item(1);
```

Comment:
Do you need the 'value' here?

Reply:
Good catch I use pair.item(1) below so can remove it.